### ISSUE TITLE
Opened panels should match the state of panels in the panels menu

### ISSUE THREAD

**Issue by bmingles** [deephaven/web-client-ui#1924]
Panels in the panels menu currently get updated whenever field update events are fired on the connection (`connection.subscribeToFieldUpdates` in `AppMainContainer`). These respond to any changes on the server not just the current session. In contrast, panels that are opened only update when commands are issued to the current console session. They do not update if a command is issued from a different session.

It would be preferred if open panels always reflected the same state as the panels in the panels menu.
1. This avoids a mismatch such that clicking on a panel in the menu results in a different state than a currently opened panel
2. This opens up some interesting options for other sessions to update panels such as IDE extensions

**Comment by mattrunyon** [comment]
The first point is intentional I believe. There are use cases where people want multiple copies of the same table but might buy filtering it differently in the UI.

**Comment by bmingles** [comment]
Correct, opened panels never update based on server side changes that aren't initiated from the current console session. e.g. running command in a different browser tab would not update the opened panel in the current tab, but if you clicked on the same name in the panels menu, it would then update with what was in the other browser tab.
